src/app/user-management/user-profile/user-profile.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="columns">
+  <ng-progress [positionUsing]="'marginLeft'" [minimum]="0.15" [maximum]="1" [speed]="200" [showSpinner]="true" [direction]="'leftToRightIncreased'"
+    [color]="'#1B95E0'" [trickleSpeed]="300" [thick]="true" [ease]="'linear'">
+  </ng-progress>
+  <div class="column is-offset-1 is-four-fifths">
+    <div class="card">
+      <header class="card-header">
+        <p class="card-header-title">
+         Profile
+        </p>
+      </header>
+      <div class="card-content">
+        <table class="table scroll is-responsive">
```

Comment:
this will not be a table; no need to show id, keep it hidden. it should be a form with disabled input for username, role etc; and will allow as soon as we will have first name, last name, contact info etc.